lib/net.js

```diff
@@ -1081,10 +1081,6 @@ function afterConnect(status, handle, req, readable, writable) {
     return;
   }
 
-  // Update handle if it was wrapped
-  // TODO(indutny): assert that the handle is actually an ancestor of old one
```

Comment:
Should this TODO be removed?

Reply:
that's why I pinged @indutny

Reply:
I'm afraid I no longer remember what this was about. This comments is years old.

Reply:
Perhaps this particular change alone can be done in a separate PR then?